collector/kernel/process_handler.h

```diff
@@ -85,51 +54,24 @@ class ProcessHandler {
 
   void pid_exit(std::chrono::nanoseconds timestamp, struct jb_agent_internal__pid_exit const &msg);
 
-  void report_task_status(std::chrono::nanoseconds timestamp, struct jb_agent_internal__report_task_status const &msg);
-
 #ifdef DEBUG_TGID
   void debug_tgid_dump();
 #endif // DEBUG_TGID
 
 private:
   struct PerPidData {
```

Comment:
This structure is left empty. Can it be removed completely?

Reply:
yes